packages/create-sitecore-jss/src/templates/nextjs-sxa/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "dependencies": {
-    "bootstrap": "^5.1.3",
```

Comment:
We probably want to leave `nextjs-sxa` alone and leave it up to the SXA team to update/test these dependencies?

Reply:
Makes sense for me, I was thinking about that